Makefile

```diff
@@ -15,7 +16,7 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 # $2 - Dockerfile path
 # $3 - context directory for image build
 # It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,origin-$(GO_PACKAGE),./Dockerfile,.)
+$(call build-image,ocp-cluster-svcat-apiserver-operator,registry.svc.ci.openshift.org/ocp/4.2:cluster-svcat-apiserver-operator,./Dockerfile,.)
```

Comment:
Don't know anything about this, so forgive me if this is a dumb question. Does the `build-image` macro had an additional argument now? Before, the third argument was the `Dockerfile` path. Now it's an image name?

If so, should we update the doc comments that describe the arguments?